test/images/agnhost/Dockerfile

```diff
@@ -17,6 +17,10 @@ FROM $BASEIMAGE
 
 CROSS_BUILD_COPY qemu-QEMUARCH-static /usr/bin/
 
+# Switch to Apline Linux Mirror of Clarkson University, the mirror system is incredibly flaky:
+# https://mirror.clarkson.edu/distributions.html#alpine
+RUN sed -i 's/http\:\/\/dl-cdn.alpinelinux.org/http\:\/\/mirror.clarkson.edu/g' /etc/apk/repositories
```

Comment:
unrelated to the PR but curious, how hard is to set up our own mirror in k8s infra?